modules/cnf-configuring-the-ptp-fast-event-publisher.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-configuring-the-ptp-fast-event-publisher_{context}"]
+= Configuring the PTP fast event notifications publisher
+
+To start using PTP fast event notifications for a network interface in your cluster, you must enable the fast event publisher in the PTP Operator `PtpOperatorConfig` custom resource (CR) and configure `ptpClockThreshold` values in a `PtpConfig` CR that you create.
+
+To configure the fast event notifications publisher, perform the following procedure.
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the PTP Operator and AMQ Interconnect Operator.
+
+.Procedure
+
+. Modify the `spec.ptpEventConfig` field of the `PtpOperatorConfig` resource and set appropriate values.
```

Comment:
```suggestion
. Modify the `spec.ptpEventConfig` field of the `PtpOperatorConfig` resource and set appropriate values by running the following command:
```

I expect that [this change](https://github.com/openshift/openshift-docs/pull/36585) might merge before your PR does.

Reply:
Noted thanks.